source/Cosmos.Core/CPU.cs

```diff
@@ -62,5 +63,12 @@ public static bool DisableInterrupts()
             mInterruptsEnabled = false;
             return xResult;
         }
+
+        public void Shutdown()

```

Comment:
@jp2masa why Reboot method is also in Cosmos.Core/CPU.cs ?
Because Cosmos can be rebooted from ACPI too.